spec/dummy/app/controllers/application_controller.rb

```diff
@@ -1,2 +1,9 @@
 class ApplicationController < ActionController::Base
+  helper_method :current_user
+
+  private
+
+  def current_user
+    AdminUser.first
```

Comment:
```suggestion
    AdminUser.where(id: ENV["CURRENT_USER_ID"]) || AdminUser.first
```

Reply:
you suggest this to make it easier to use the dummy app ?